### ISSUE TITLE
I can't reproduce the results in the paper

### ISSUE THREAD

**Issue by ClarissaDHL** [justchenhao/STANet#51]
Hi. I can't reproduce the results in the paper. I wonder if there is a problem with my training strategy or  authors use selected slice images for training rather than slice images.
And I got a question that u use two independent optimizers. Is this for learning rate setting considerations or for faster convergence?

**Comment by nmxian** [comment]
I run the code and  can not get the results in the paper too, have you find the reason?

> Hi. I can't reproduce the results in the paper. I wonder if there is a problem with my training strategy or authors use selected slice images for training rather than slice images.
> And I got a question that u use two independent optimizers. Is this for learning rate setting considerations or for faster convergence?

**Comment by Perfect-You** [comment]
Metoo, and this is the results of my reproduction:
BASE   precision 63.5       recall 95.5          f1_score 76.3
BAM   precision 65.9       recall 94.6          f1_score 77.7
PAM   precision 77.0        recall 89.2          f1_score 82.7

**Comment by weizhiliang0520** [comment]
> Metoo, and this is the results of my reproduction: BASE precision 63.5 recall 95.5 f1_score 76.3 BAM precision 65.9 recall 94.6 f1_score 77.7 PAM precision 77.0 recall 89.2 f1_score 82.7

Hi, I downloaded the github code, kept the original settings, and ran python train.py. But there are many errors, could you please send me your code? Thank you!

**Comment by weizhiliang0520** [comment]
> Metoo, and this is the results of my reproduction: BASE precision 63.5 recall 95.5 f1_score 76.3 BAM precision 65.9 recall 94.6 f1_score 77.7 PAM precision 77.0 recall 89.2 f1_score 82.7

weizhiliang0520@gmail.com
